app/components/container-row/template.hbs

```diff
@@ -25,10 +25,10 @@
 {{#if showStats}}
   <td class="spark-td">
     {{#if model.cpuSpark}}
-      {{spark-line data=model.cpuSpark     width=60 height=15 max=cpuMax     prefix="CPU: " formatter="percent"}}
-      {{spark-line data=model.memorySpark  width=60 height=15 max=memoryMax  prefix="Memory: " formatter="mib"}}
-      {{spark-line data=model.networkSpark width=60 height=15 max=networkMax prefix="Network: " formatter="kbps"}}
-      {{spark-line data=model.storageSpark width=60 height=15 max=storageMax prefix="Storage: " formatter="kbps"}}
+      {{spark-line data=model.cpuSpark     width=80 height=30 max=cpuMax     prefix="CPU: " formatter="percent" type="cpu"}}
```

Comment:
I used RDM to change my screen res to 1400x900 non-HiDPI mode. They look a little pixelated. I'll change it back.